From 3808257bfc402bc170581f2151959846d80996be Mon Sep 17 00:00:00 2001
From: Ashokkumar G <0xfee1dead.sa@gmail.com>
Date: Mon, 26 Jan 2015 11:08:19 -0700
Subject: [PATCH 4/9] drivers:spi: Adding mmp dependency for SPI_PXA2XX

adding arch_mmp dependency for SPI_PXA2XX as pxa168 based gplugd
uses SPI_PXA2XX

resolving compilation error of multiple definitions of ssp functions
pxa_ssp_request, pxa_ssp_free from plat-pxa/ssp.c and include/linux/pxa2xx_ssp.h

Signed-off-by: Ashokkumar G <0xfee1dead.sa@gmail.com>
---
 drivers/spi/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/spi/Kconfig b/drivers/spi/Kconfig
index 84e7c9e6ccef..caa969f64ff4 100644
--- a/drivers/spi/Kconfig
+++ b/drivers/spi/Kconfig
@@ -383,8 +383,8 @@ config SPI_PXA2XX_DMA
 
 config SPI_PXA2XX
 	tristate "PXA2xx SSP SPI master"
-	depends on (ARCH_PXA || PCI || ACPI)
-	select PXA_SSP if ARCH_PXA
+	depends on (ARCH_PXA || ARCH_MMP || PCI || ACPI)
+	select PXA_SSP if (ARCH_PXA || ARCH_MMP)
 	help
 	  This enables using a PXA2xx or Sodaville SSP port as a SPI master
 	  controller. The driver can be configured to use any SSP port and
-- 
2.14.1

